integration_test.go

```diff
@@ -34,7 +47,8 @@ func prepareVault() {
 	if err != nil {
 		log.Fatalf("Error in initVaultDev.sh %v", err)
 	}
-	cmd := exec.Command("./vault", "read", "-field=role_id", "auth/approle/role/my-role/role-id")
+    vaultBin := "./test-files/vault/vault" + vaultVersion
+	cmd := exec.Command(vaultBin, "read", "-field=role_id", "auth/approle/role/my-role/role-id")
```

Comment:
↑ Here is an example of inconsistent indentation.

Reply:
Yeah...tabs vs spaces...on it